### DISCUSSION TITLE
[docs] Add a mention in the docs for adding @types/bun to the compilerOptions.types array of the tsconfig.json

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [oven-sh/bun#10471] [Ideas]
Hello! 🔥🤩👋🐰🍞

I searched in Google and in the docs, and there's no mention of adding `@types/bun` to the `compilerOptions.types` array of the `tsconfig.json`. I get an error if I don't add it, and also don't get types completion.
> Cannot find name 'Bun'. Do you need to install type definitions for Bun? Try `npm i --save-dev @types/bun` and then add 'bun' to the types field in your tsconfig.

Even the `bun init` generated `tsconfig.json` doesn't have it!

**Comment by MERakram** [comment]
This is minor but it can't be used without it
THX!
